src/components/TRPCDemo/TRPCDemo.tsx

```diff
@@ -0,0 +1,53 @@
+import { trpcReact } from 'Providers/TRPCProvider'
+import { useState } from 'react'
+
+export function TRPCDemo() {
+  return (
+    <>
+      <SimpleGetter />
+      <Subscription />
+      <Accounts />
+    </>
+  )
+}
+
+function SimpleGetter() {
+  const { data, isLoading, isError, error } = trpcReact.greeting.useQuery({
+    name: 'Electron',
+  })
+
+  if (isLoading) {
+    return <h1>Loading...</h1>
+  }
+
+  if (isError) {
+    return <h1>Error: {error?.message}</h1>
+  }
+
+  if (!data) {
+    return null
+  }
+
+  return <h1>{data.text}</h1>
+}
```

Comment:
Here I wanted to show how the tRPC client uses react-query, so all procedure calls come with built in loading/error/pending/etc states, which is super nice. We wouldn't have to use stuff like the custom `useAsyncDataWrapper` anymore.